src/unix/fs.c

```diff
@@ -889,45 +889,6 @@ static ssize_t uv__fs_write(uv_fs_t* req) {
 }
 
 static ssize_t uv__fs_copyfile(uv_fs_t* req) {
-#if defined(__APPLE__) && !TARGET_OS_IPHONE
```

Comment:
We should be able to remove this now too: https://github.com/libuv/libuv/blob/94043e2fbe5c9ad2c955b2ec012b20b51323296b/src/unix/fs.c#L63

Reply:
done!